Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,8 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+    return <img id='banner' src='https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png'></img>
```

Comment:
there is no need for a closing </img> tag because img is one of the special "self-closing" html elements.
you can find some more info about that here >> http://xahlee.info/js/html5_non-closing_tag.html